LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
Is there a reason for GPLv3 vs GPLv2?  I think most of backdrop is GPLv2 but if Tawk has reasons for using GPLv3 then can't see a problem as it specified GPLv2 or later.

Reply:
We've been using GPL 3 licensing for most of our plugins so it's better to have this module align with them.